package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@fabric-ds/css",
   "description": "Fabric CSS core components and utilities",
-  "version": "1.1.9",
+  "version": "1.1.10-next.1",
```

Comment:
I don't understand why the version number includes `next` now that it's released to main.